examples/scripts/blender/sdf_exporter.py

```diff
@@ -1,235 +1,217 @@
-import xml.etree.ElementTree as ET
-from os import path
-from xml.dom import minidom
-
 import bpy
-from bpy.props import StringProperty
-from bpy.types import Operator
+import os.path
 from bpy_extras.io_utils import ImportHelper
+from bpy.props import StringProperty, BoolProperty
+from bpy.types import Operator
 
-# Tested Blender version: 2.82/3.2
+import xml.etree.ElementTree as ET
+from xml.dom import minidom
 
-########################################################################################################################
-### Exports model.dae of the scene with textures, its corresponding model.sdf file, and a default model.config file ####
-########################################################################################################################
 def export_sdf(prefix_path):
-
-    dae_filename = "model.dae"
-    sdf_filename = "model.sdf"
-    model_config_filename = "model.config"
-    lightmap_filename = "LightmapBaked.png"
-    model_name = "my_model"
-    meshes_folder_prefix = "meshes/"
-
+    dae_filename = 'model.dae'
+    sdf_filename = 'model.sdf'
+    model_config_filename = 'model.config'
+    lightmap_filename = 'LightmapBaked.png'
+    model_name = 'my_model'
+    meshes_folder_prefix = 'meshes/'
+    
     # Exports the dae file and its associated textures
-    bpy.ops.wm.collada_export(
-        filepath=path.join(prefix_path, meshes_folder_prefix, dae_filename),
-        check_existing=False,
-        filter_blender=False,
-        filter_image=False,
-        filter_movie=False,
-        filter_python=False,
-        filter_font=False,
-        filter_sound=False,
-        filter_text=False,
-        filter_btx=False,
-        filter_collada=True,
-        filter_folder=True,
-        filemode=8,
-    )
+    bpy.ops.wm.collada_export(filepath=prefix_path+meshes_folder_prefix+dae_filename, 
```

Comment:
Can we keep the `path.join` here? That is preferred over using `+` for paths. And since there aren't any other changes, could you revert this to the previous version to reduce the diff as much as possible.